testool/docker/lllc/patch.diff

```diff
@@ -0,0 +1,15 @@
+diff --git a/liblll/Parser.cpp b/liblll/Parser.cpp
+index 3b68bc2da..06a3eb067 100644
+--- a/liblll/Parser.cpp
++++ b/liblll/Parser.cpp
+@@ -67,8 +67,8 @@ void dev::lll::debugOutAST(ostream& _out, sp::utree const& _this)
+ 
+ 		break;
+ 	case sp::utree_type::int_type: _out << _this.get<int>(); break;
+-	case sp::utree_type::string_type: _out << "\"" << _this.get<sp::basic_string<boost::iterator_range<char const*>, sp::utree_type::string_type>>() << "\""; break;
+-	case sp::utree_type::symbol_type: _out << _this.get<sp::basic_string<boost::iterator_range<char const*>, sp::utree_type::symbol_type>>(); break;
++	// case sp::utree_type::string_type: _out << "\"" << _this.get<sp::basic_string<boost::iterator_range<char const*>, sp::utree_type::string_type>>() << "\""; break;
++	// case sp::utree_type::symbol_type: _out << _this.get<sp::basic_string<boost::iterator_range<char const*>, sp::utree_type::symbol_type>>(); break;
```

Comment:
nitpick
```suggestion
```

Reply:
The intention is to remove the commented-off code from the patch. Keeping only the deletion part in the patch should do the trick.